inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -162,10 +162,8 @@ def _(self, simulator: SWASH, input_dir):  # pylint: disable=unused-argument
     bathymetry_x_num = len(self.bathymetry.x_uniques())
     bathymetry_y_num = len(self.bathymetry.y_uniques())
 
-    bathymetry_x_delta = (self.bathymetry.x_range[1] -
-                          self.bathymetry.x_range[0]) / bathymetry_x_num
-    bathymetry_y_delta = (self.bathymetry.y_range[1] -
-                          self.bathymetry.y_range[0]) / bathymetry_y_num
+    bathymetry_x_delta = (self.bathymetry.x_ptp()) / bathymetry_x_num
```

Comment:
What is this x_ptp?

Reply:
It gets the difference between maximum and minimum values of $x$, in this case. It follows the nomenclature of NumPy (see [this](https://numpy.org/doc/stable/reference/generated/numpy.ptp.html)), for which ptp refers to "peak-to-peak" value.